model/registry/profile.yaml

```diff
@@ -0,0 +1,102 @@
+groups:
+
+  - id: registry.profile.build_id
+    type: attribute_group
+    prefix: profile.build_id
+    brief: >
+      Unique identifier for an executable.
+    attributes:
+      - id: gnu
+        stability: experimental
+        type: string
+        brief: >
+          The GNU build ID as can be found in the `.note.gnu.build-id` ELF section.
+        examples: ['c89b11207f6479603b0d49bf291c092c2b719293']
+      - id: go
+        stability: experimental
+        type: string
+        brief: >
+          The Go build ID.
+        examples: ['foh3mEXu7BLZjsN9pOwG/kATcXlYVCDEFouRMQed_/WwRFB1hPo9LBkekthSPG/x8hMC8emW2cCjXD0_1aY']
+
+  - id: registry.profile.link
+    type: attribute_group
+    prefix: profile.link
+    brief: >
+      Connect Profile information to Spans and Traces.
+    attributes:
+      - id: trace_id
```

Comment:
@open-telemetry/specs-semconv-approvers thoughts?

* `trace.id` is in line with the direction we seem to be heading: https://github.com/open-telemetry/semantic-conventions/issues/1118#issuecomment-2159136053
* `trace_id` matches the logging spec's [Trace Context in Legacy Formats](
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/README.md#trace-context-in-legacy-formats) (see https://github.com/open-telemetry/opentelemetry-specification/issues/3303)

Reply:
Today, we have `trace_id` in legacy formats for logs and the corresponding OTLP-native field.

So, `trace.id` would be a new (yet another representation) of the same thing. That's why I'm slightly more in favor of  `trace_id`. Also, `trace.id` would make sense if we expect more attributes under the `trace` namespace, which I think is not the case here.

Reply:
Whatever we choose let's make sure it is consistent with what we do for logs. For now that means using `trace_id`/`span_id`.

Reply:
Removed the attributes around span/trace ID as there is a dedicated field in the OTel profiling signal.